website/docs/guides/container_service_extension.html.markdown

```diff
@@ -0,0 +1,438 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: Container Service Extension"
+sidebar_current: "docs-vcd-guides-cse"
+description: |-
+Provides guidance on Container Service Extension for VCD.
```

Comment:
We need to clarify it. It should not be "guidance on CSE", but "a guide to deploy ...". I.e. we need more specificity :)

Reply:
Fixed!